packages/ui/Marquee/MarqueeController.js

```diff
@@ -135,8 +137,23 @@ const MarqueeController = hoc(defaultConfig, (config, Wrapped) => {
 			};
 		}
 
+		componentDidMount () {
+			if (platform.webos) {
```

Comment:
@teckliew + @rundmt wdyt of removing these checks?

Seems like it would unnecessarily restrict handling the event. It complicates testing in Chrome (though you can simulate the webos UA string) but more importantly would preclude this from working on another platform just to avoid setting a key handler. We could avoid that overhead by using `core/dispatcher` which will manage setting the single listener on `document`.

Reply:
Yeah I don't mind removing them.

Reply:
Sounds good. `pointerHide` should take care of the platform specific handling.